hack/verify-receipts-upgrade-migration.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script tests "krew system receipts-migration" which was introduced for
+# migrating krew 0.2.x to krew 0.3.x.
+#
+# TODO(ahmmetb,corneliusweig) remove at/after krew 0.4.x when receipts-migration
+# is no longer supported.
+
+set -euo pipefail
+
+[[ -n "${DEBUG:-}" ]] && set -x
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BINDIR="${SCRIPTDIR}/../out/bin"
+goos="$(go env GOOS)"
+goarch="$(go env GOARCH)"
+
+install_krew_0_2_1() {
+    # from https://github.com/kubernetes-sigs/krew/blob/v0.2.1/README.md
+    krew_root="${1}"
+    cd "$(mktemp -d)"
+    curl -fsSLO "https://storage.googleapis.com/krew/v0.2.1/krew.{tar.gz,yaml}"
+    tar zxf krew.tar.gz
+    env KREW_ROOT="${krew_root}" ./krew-"${goos}_amd64" install \
+        --manifest="./krew.yaml" --archive="./krew.tar.gz"
```

Comment:
nit: it would work with just the `--manifest` option, but no need to change.

Reply:
I could just wget the manifest from krew-index repo and install with that but wanted to stick to what users do.